services/tracing.py

```diff
@@ -0,0 +1,49 @@
+"""Functions to configure OTel tracing and instrumentation
+We use OpenTelemetry (OTel) to instrument the app and capture telemetry
+data about requests in traces and spans emitted by our code. We send
+this data to Honeycomb to view and interrogate it, helping us to
+understand the internal state of our system and assist our users.
+
+In production, the functions below are invoked in
+deploy/gunicorn/conf.py. OTel requires environment variables to be set;
+these are described in dotenv-sample and DEPLOY.md."""
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+def set_up_tracing():
+    """Setup tracing infrastructure for production.
+    The resource represents the entity producing telemetry data.
```

Comment:
```suggestion
    """Setup tracing infrastructure for production.
    
    The resource represents the entity producing telemetry data.
```
Clearer to have an empty line between the summary line and the further detail.